roles/kubernetes/master/defaults/main.yml

```diff
@@ -52,6 +52,7 @@ kube_apiserver_admission_control:
       {%- if kube_version | version_compare('v1.9', '<') -%}
       GenericAdmissionWebhook
       {%- else -%}
+      MutatingAdmissionWebhook
```

Comment:
Jinja wont render the manifest template properly with this approach. You'll end up with a space between initializers (instead of comma separated) and the API server will fail to start. One fix is putting them on the same line MutatingAdmissionWebhook,ValidatingAdmissionWebhook